dht_test.go

```diff
@@ -805,7 +805,7 @@ func TestEmptyTable(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	nDHTs := 50 // needs more than 40 peers so we don't add all of them to our routing table.
```

Comment:
@Stebalien Why do we have this assumption ?

Reply:
I can't remember but I was probably thinking something along the lines of: Given 50 peers, about half of them will differ in the first bit. That means we'll have 25 candidates for the first bucket, leaving 5 peers out.

But there's still a high probability that this _won't_ be the case.